CHANGELOG-ABP.md

```diff
@@ -0,0 +1,23 @@
+# Release Notes
+## Chromium 67.0
+* Updated to Chromium `67.0.3396.87`
+* [Update](https://gitlab.com/eyeo/adblockplus/chromium/issues/7) the way `libadblockplus` and `libadblockplus-android` are built
+* [Added support](https://issues.adblockplus.org/ticket/6632) for:
+   - `am-ET - Amharic (Ethiopia)`
+   - `fr-CA - French (Canada)`
+   - `fil-PH - Filipino (Philippines)`
+   - `sw-KE - Kiswahili (Kenya)`
+* [Initialize](https://gitlab.com/eyeo/adblockplus/chromium/issues/9) filter engine asynchronously, and [wait](https://gitlab.com/eyeo/adblockplus/chromium/issues/14) for it in Settings.
+* Stop using [deprecated libadblockplus-android API](https://gitlab.com/eyeo/adblockplus/chromium/issues/13)
+* [Disable](https://gitlab.com/eyeo/adblockplus/chromium/issues/20) Chromium's built-in ad blocking
+* [Support](https://gitlab.com/eyeo/adblockplus/chromium/issues/22) configuring of android package name filter list network requests 
+* [Correctly detect](https://gitlab.com/eyeo/adblockplus/chromium/issues/24) the type of image resources
+
+[Full list of `libadblockplus` changes](https://github.com/adblockplus/libadblockplus/compare/ea5309a0a6f3c5ab1e378b79c09d930ac3fbcfd0...40f5d4d2d00abe3f94ce69210267bcce908cd748).
+
```

Comment:
i think it makes sense to mention explicitly  that build machine now requires some packages to be installed (for 7zip). Makes sense?

Reply:
Makes sense. Mentioned that explicitly, thanks.